explainer.md

```diff
@@ -827,6 +828,20 @@ facts were considered by our decision process.
   [WriteOptions.sync option](https://github.com/google/leveldb/blob/master/doc/index.md#synchronous-writes).
 
 
+### Default Bucket Quota
+
+A default quota will be assigned to every storage bucket that is created
+with `openOrCreate()` without a `quota` policy. The behavior of the
+default bucket quota will be user agent specific.
+
+In Chrome, we plan to have the default quota for a storage bucket to
```

Comment:
Let's avoid saying "we" to mean Chrome engineers. This doc will hopefully gain non-Chrome authors.

In Chrome, we plan to have -> Chrome plans to have
we think -> the Chrome team thinks

Reply:
Sorry :( I keep on forgetting. 
Thanks for catching this.